crates/astria-conductor/src/executor.rs

```diff
@@ -114,6 +118,12 @@ struct Executor<C> {
     /// so that we can mark the block as final on the execution layer when
     /// we receive a finalized sequencer block.
     sequencer_hash_to_execution_hash: HashMap<Vec<u8>, Vec<u8>>,
+
+    /// most recently executed sequencer block hash
+    last_safe_block_hash: Hash,
```

Comment:
this block isn't necessarily safe (unless the comment needs to be updated?)

Reply:
the intention is for this to be the most recent safe block. i'll update the comment

Reply:
i had to change this to the most recently executed sequencer block hash. i think we should be able to have this be more clear once #205 get's merged.

Reply:
made #220 to take a closer look as this